modules/server/templates/configs/forseti_conf_server.yaml.tpl

```diff
@@ -301,6 +301,10 @@ notifier:
                 data_format: csv
                 # gcs_path should begin with "gs://"
                 gcs_path: gs://${FORSETI_BUCKET}/scanner_violations
+            - name: slack_webhook
+              configuration:
+                data_format: json  # slack only supports json
+                webhook_url: ${VIOLATIONS_SLACK_WEBHOOK}
```

Comment:
By default, this will render as

`webhook_url: ""`

Do we know if Forseti correctly handles that case and ignores the webhook entirely? @hshin-g

Reply:
Forseti has this code to handle the empty url.

    def run(self):
        """Run the slack webhook notifier"""
        if not self.notification_config.get('webhook_url'):
            LOGGER.warn('No url found, not running Slack notifier.')
            return

but you're right. it would make sense to omit the slack notifier so this warning isn't logged. I'll make that change.